WORKING_GROUPS.md

```diff
@@ -29,9 +29,13 @@ back in to the TSC.
 * [Addon API](#addon-api)
 * [Benchmarking](#benchmarking)
 * [Post-mortem](#post-mortem)
+* [Intl](#Intl)
+* [HTTP](#HTTP)
```

Comment:
Hm, I just tried this link in my browser and it didn't work. Lowercase `http` worked fine though. Same for Intl.